### PR TITLE
Translation mapping presets

### PR DESCRIPTION
**Pull request by phlax**
Similar to the old project treestyle presets, this PR add presets to the fs ui translation mapping widget

### CODE REVIEW

File: pootle/apps/pootle_fs/forms.py [hunk 1]
```diff
@@ -18,6 +18,13 @@
     fs_plugins, fs_translation_mapping_validator, fs_url_validator)
 
 
+FS_CHOICES = (
+    ("gnu", "GNU-style", "/po/<language_code>.<ext>"),
+    ("non-gnu", "non GNU-style", "/<language_code>/<dir_path>/<filename>.<ext>"),
+    ("django", "Django-style", "/locale/<lang_code>/LC_MESSAGES/<filename>.<ext>"),
+    ("custom", "Custom"))
```
Comment (dwaynebailey) [inline pootle/apps/pootle_fs/forms.py]: I'd say these should be localizable, not sure if it needs `lazy_gettext`

File: pootle/apps/pootle_fs/forms.py [hunk 2]
```diff
@@ -30,9 +41,18 @@ class ProjectFSAdminForm(forms.Form):
...
+        required=False,
+        choices=(
+            [("", "-----"), ]
+            + [(x[0], x[1]) for x in FS_CHOICES]),
+        widget=forms.Select(
+            attrs={'class': 'js-select2 js-select-fs-mapping'}))
     translation_mapping = forms.CharField(
         help_text=_(
-            "The translation mapping for your filesystem"))
+            "The translation mapping for your filesystem"),
```
Comment (unho) [inline pootle/apps/pootle_fs/forms.py]: Nit: I think this fits in the previous line.

**Comment by unho** [comment]
Nits aside lgtm. Is there any live server to test?

File: pootle/apps/pootle_fs/forms.py [hunk 3]
```diff
@@ -18,6 +18,17 @@
     fs_plugins, fs_translation_mapping_validator, fs_url_validator)
 
 
+FS_CHOICES = (
+    ("gnu", _("GNU-style"), "/po/<language_code>.<ext>"),
```
Comment (phlax) [inline pootle/apps/pootle_fs/forms.py]: i think as its an example - lets use `po` - then users can use custom if they need to change

**Pull request merged**